apps/frontend/src/shared/i18n/locales/en/roadmap.json

```
@@ -0,0 +1,204 @@
+    "impact": "Impact",
+    "competitorInsightTooltip": "This feature addresses competitor pain points",
+    "insight": "Insight",
+    "goToTask": "Go to Task"
+  },
+  "deleteDialog": {
+    "title": "Delete Feature?",
+    "description": "This will permanently remove \"{{title}}\" from your roadmap.",
+    "cancel": "Cancel",
+    "delete": "Delete"
+  },
+  "complexity": {
```

Biome (2.1.2)

[error] The key complexity was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)